sources/packages/web/src/components/common/ScholasticStandingForm.vue

```diff
@@ -34,9 +47,23 @@ interface ScholasticStandingBeforeSubmission
 export default defineComponent({
   emits: ["submit", "cancel"],
   props: {
+    scholasticStandingId: {
+      type: Number,
+      required: true,
```

Comment:
Is should not be required since the component can be loaded with the `initialData`.